addons/mail/static/tests/discuss/call/call.test.js

```diff
@@ -50,15 +50,21 @@ test("basic rendering", async () => {
     await contains(".o-discuss-CallParticipantCard[aria-label='Mitchell Admin']");
     await contains(".o-discuss-CallActionList");
     await contains(".o-discuss-CallMenu-buttonContent");
-    await contains(".o-discuss-CallActionList button", { count: 6 });
+    await contains(".o-discuss-CallActionList button", { count: 7 });
     await contains("button[aria-label='Unmute'], button[aria-label='Mute']"); // FIXME depends on current browser permission
-    await contains(".o-discuss-CallActionList button[aria-label='Deafen']");
+    await contains("button[aria-label='Voice Settings']");
     await contains(".o-discuss-CallActionList button[aria-label='Turn camera on']");
+    await contains("button[aria-label='Video Settings']");
     await contains(".o-discuss-CallActionList button[aria-label='Share Screen']");
     await contains("[title='More']");
     await contains(".o-discuss-CallActionList button[aria-label='Disconnect']");
     await click("[title='More']");
+    await contains(".o-dropdown-item", { count: 2 });
     await contains(".o-dropdown-item:contains('Raise Hand')");
+    await contains(".o-dropdown-item:contains('Disable speaker autofocus')");
+    await contains(".o-discuss-Call-layoutActions");
```

Comment:
I believe this `contains` is useless? Since it's been tested again on the next line.